tests/unit/test_utils/test_signature.py

```diff
@@ -156,16 +159,29 @@ def test_add_types_to_signature_namespace() -> None:
     assert ns == {"int": int, "str": str}
 
 
-def test_add_types_to_signature_namespace_with_existing_types() -> None:
+def test_add_types_to_signature_namespace_no_warn() -> None:
     """Test add_types_to_signature_namespace with existing types."""
-    ns = add_types_to_signature_namespace([str], {"int": int})
-    assert ns == {"int": int, "str": str}
+    if os.environ.get("LITESTAR_WARN_SIGNATURE_NAMESPACE_OVERRIDE", None):
+        del os.environ["LITESTAR_WARN_SIGNATURE_NAMESPACE_OVERRIDE"]
```

Comment:
```suggestion
    monkeypatch.delenv("LITESTAR_WARN_SIGNATURE_NAMESPACE_OVERRIDE", raising=False)
```

So it's resets things properly. Easier to reason about IMO